src/lib/common/statistics.h

```diff
@@ -315,9 +316,9 @@ extern int noOfSimulatedNotifications;
 
 /* ****************************************************************************
 *
-* timingStatistics - 
+* renderTimingStatistics -
 */
-extern std::string timingStatistics(std::string indent, Format format, std::string apiVersion);
+extern std::string renderTimingStatistics(void);
```

Comment:
Yes, pretty printer has been removed in this PR, leading to a cleaner code (i.e. no more "ending comma calculations"). 

I don't think thi is an issue, as you can always pipe your favourity beautifier if you need the output in that way.